addon/components/o-s-s/empty-state.hbs

```diff
@@ -0,0 +1,22 @@
+<div class="fx-1 fx-col fx-xalign-center fx-gap-px-18">
+  {{#if (has-block "image")}}
+    <span class="">
+      {{yield to="image"}}
+    </span>
+  {{else if @image}}
+    <OSS::Badge @icon={{@image}} @size="lg" />
+  {{/if}}
+  <div class="fx-col fx-xalign-center fx-gap-px-6">
+    <div class="font-color-gray-900 font-size-{{this.titleSize}} font-weight-semibold">
+      {{@title}}
+    </div>
+    <div class="font-color-gray-500 font-size-{{this.subtitleSize}}">
+      {{@subtitle}}
+    </div>
```

Comment:
`title` & `subtitle` are marked as optional in the TS but there is no check on their presence here, leading to empty divs.

I think we can remove the optional side in the TS and make both mandatory for now (i don't remember seeing an empty state w/ only a subtitle, or only a title)

Also, i think the `div` can be `span` instead.